sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -307,6 +312,10 @@ def queue_state(self):
         while True:
             try:
                 out, err, retval = self.system_call(system_command)
+                if retval != 0:
+                    logging.warning(self._system_call_error_warn_msg(system_command))
+                    time.sleep(gs.WAIT_PERIOD_QSTAT_PARSING)
+                    continue
```

Comment:
But this here is not what was discussed? What we discussed should be sth like this here:
```suggestion
                    raise subprocess.CalledProcessError(self._system_call_error_warn_msg(system_command))
```
Together with the other changes in `task_state`, as it was discussed.